src/routes/healthcheck/+server.ts

```diff
@@ -0,0 +1,3 @@
+export async function GET() {
+	return new Response("OK", { status: 200 });
```

Comment:
It could be interesting to check the DB connectivity here and other dependencies (if there are)